create-keystone-app/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "create-keystone-next-app",
+  "name": "create-keystone-app",
   "version": "1.2.1",
```

Comment:
We're going to have to manually change the version number here to match what's currently in the `keystone-5` repo. If we don't do that we'll end up publishing into a version range that already exists.

Reply:
Updated to `3.5.2`